rootfs/usr/share/applications/org.chimeraos.Gamescope.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Return to Gaming Mode
+Exec=/usr/bin/steamos-session-select gamescope
+Icon=/usr/share/icons/chimeraos/scalable/actions/return-to-game-mode.svg
```

Comment:
I think using the Steam icon is probably better?

Reply:
We need to add it. Can be done. But maybe we should have a broader discussion about branding the OS?

Reply:
I'll merge now, we can figure out the icon later. It should be changed to something else in any case, it is too big/detailed to be used as an icon IMO.

Reply:
> I'll merge now, we can figure out the icon later. It should be changed to something else in any case, it is too big/detailed to be used as an icon IMO.

True that. Maybe we can do a simplified version of the lion head to be iconified (on small screens it doesn't look right)